### PR TITLE
docs: minor fixes

### PR DESCRIPTION
**Pull request by mmarinchenko**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -959,8 +960,7 @@ For example, on Windows, setting `WasmShellMonoTempFolder` to `C:\MonoWasmSDKs`,
 instead of `C:\Users\xxx\AppData\Local\Temp\mono-wasm-e351637985e`.
 
 ### Windows Long Path support
-The bootstrapper supports Windows 10 long paths by default, but there may be cases where the 
-[`\\?\` path format](https://blogs.msdn.microsoft.com/jeremykuhne/2016/06/21/more-on-new-net-path-handling/) may not be supported.
```
Comment (mmarinchenko) [inline Readme.md:963]: The MSDN link is broken. I didn't find original article.

Comment (jeromelaban) [reply Readme.md:963]: Indeed, searching if we can get a better url.

Comment (jeromelaban) [reply Readme.md:963]: Let's rely on wayback machine for now.

**Review by jeromelaban — APPROVED** [review]

**Comment by jeromelaban** [comment]
Thanks for your contribution!

**Pull request merged**